requirements.txt

```diff
@@ -7,4 +7,3 @@ gunicorn==18.0
 itsdangerous==0.23
 -e git://github.com/kennethreitz/pyandoc.git@04cd95d3426c7665fc88a1774709e7b68fe226d7#egg=pyandoc-master
```

Comment:
If we're not using pandoc, let's remove this requirement.

Reply:
Oops, sorry for the oversight! I've updated the PR.